pallets/gear/src/tests.rs

```diff
@@ -876,7 +876,7 @@ fn block_gas_limit_works() {
 
     init_logger();
     new_test_ext().execute_with(|| {
-        let remaining_weight = 791822425 + 6228060 - 1; // calc gas pid1 + pid2 - 1
+        let remaining_weight = 1080476846 + 41640740 - 1; // calc gas pid1 + pid2 - 1
```

Comment:
here better to also calculate it properly

Reply:
our calculate gas works great now

Reply:
test is ignored now